kinbot/reaction_generator.py

```diff
@@ -256,7 +256,7 @@ def generate(self):
                         for j, st_pt_j in enumerate(obj.products):
                             if st_pt_i.chemid == st_pt_j.chemid and i < j:
                                 obj.products[j] = obj.products[i]
-
+                    '''
```

Comment:
I'm adding in the try exception currently. I'll push once the test is done running.